### ISSUE TITLE
query length (PATO:0000122) and related terms

### ISSUE THREAD

**Issue by mah11** [pato-ontology/pato#172]
In PATO, PATO:0001334 ! diameter is_a PATO:0000122 ! length. This causes undesirable inferences for fission yeast cell shape phenotypes in FYPO.

More generally, it looks like PATO isn't internally consistent in how it handles terms in the PATO:0001708 ! 1-D extent branch. 

1. The text definition of length is so broad that I can't easily see the distinction between it and its superclass 1-D extent.

2.- The fact that PATO:0000122, PATO:0000119 ! height; PATO:0000915 ! thickness, PATO:0000921 ! width, and PATO:0001595 ! depth are all separate (siblings under PATO:0001708) seems inconsistent with the breadth of the PATO:0000122 text definition. 

3. Between issues 1 and 2, it's as though a broader and a narrower sense of "length" are being conflated - the broader corresponding to the text def, and the narrower as in length as opposed to width or depth.

4. It's also a tiny bit confusing that there are five terms for measurements that each correspond to one of three possible dimensions in space but that's a smaller problem than the others.

5. PATO:0001711 ! perimeter is also is_a PATO:0000122, but its definition doesn't really fit the text definition of length. A perimeter isn't a distance between two separate points; it's the distance travelled going around a 2-D shape from one point back to the starting point.

**tl;dr:** It would help us immensely if PATO:0001334 ! diameter and PATO:0001711 ! perimeter could be moved to is_a PATO:0001708 ! 1-D extent. That's the minimal change to relieve us of dodgy inferences.

Additional changes to text definitions would help clarify distinctions among the various subclasses of PATO:0001708 ! 1-D extent, but aren't so urgent.

**Comment by nicolevasilevsky** [comment]
any objections to reclassifying PATO:0001334 ! diameter and PATO:0001711 ! perimeter as children of PATO:0001708 ! 1-D extent?

If not, I'll make these changes next week.

**Comment by dosumis** [comment]
Fixed (see linked pull request)
